src/components/classResources.tsx

```diff
@@ -0,0 +1,13 @@
+import { Class } from "../types/userList";
```

Comment:
This is not a component. Create a file helpers.ts in src directory and move this function into that file.